README.md

```diff
@@ -1,40 +1,28 @@
-# ansible-role-domain-manager #
+# ansible-role-domain-manager 🐑 #
 
 [![GitHub Build Status](https://github.com/cisagov/ansible-role-domain-manager/workflows/build/badge.svg)](https://github.com/cisagov/ansible-role-domain-manager/actions)
 [![Total alerts](https://img.shields.io/lgtm/alerts/g/cisagov/ansible-role-domain-manager.svg?logo=lgtm&logoWidth=18)](https://lgtm.com/projects/g/cisagov/ansible-role-domain-manager/alerts/)
 [![Language grade: Python](https://img.shields.io/lgtm/grade/python/g/cisagov/ansible-role-domain-manager.svg?logo=lgtm&logoWidth=18)](https://lgtm.com/projects/g/cisagov/ansible-role-domain-manager/context:python)
 
-This is a skeleton project that can be used to quickly get a new
-[cisagov](https://github.com/cisagov) Ansible role GitHub project
-started.  This skeleton project contains
-[licensing information](LICENSE), as well as
-[pre-commit hooks](https://pre-commit.com) and
-[GitHub Actions](https://github.com/features/actions) configurations
-appropriate for an Ansible role.
+This is a project to provision a Domain Manager server.
+Setup: [GhostWriter wiki](https://ghostwriter.wiki/documentation/installation/first-server-run)
 
 ## Requirements ##
 
-None.
+This role uses the `git` Ansible module.
 
 ## Role Variables ##
 
-None.
+Look at [vars/main.yml](vars/main.yml) for variables that
+need to be changed for your environment.
 
 ## Dependencies ##
 
-None.
+This project has the following dependencies:
 
-## Example Playbook ##
+* [ansible-role-docker](https://github.com/cisagov/ansible-role-docker)
 
-Here's how to use it in a playbook:
-
-```yaml
-- hosts: all
-  become: yes
-  become_method: sudo
-  roles:
-    - skeleton
-```
+* [ansible-role-pip](https://github.com/cisagov/ansible-role-pip)
```

Comment:
I see you used to install docker and pip, but then switched to the cisagov ansible-roles. However, I do not see ansible-role-pip in the `meta/main.yml` dependencies or the molecule `requirements.yml` file. If this is intentional this should be removed from the README dependencies. If it is for future use I would prefer it is added when those changes are so that it is cohesive.

Reply:
The ansible-role-pip is a nested dependency since the ansible-role-docker depends on it. I can remove the bullet from the README if it's confusing.

Reply:
Yes, I would remove the bullet mentioning the `ansible-role-pip` dependency since it is a dependency of `ansible-role-docker`.  Though technically, you don't need to list `ansible-role-docker` as a dependency either, since it is automatically pulled in by this role.  "Dependency" typically implies something that needs to be present before this role can be successfully run.

Reply:
Got it. Updated the dependency section to reflect the suggested changes.